tests/unittests/benchmark/task/profet/test_profet_task.py

```diff
@@ -0,0 +1,375 @@
+""" Utilities used to test the different subclasses of the ProfetTask. """
+from pathlib import Path
+from typing import ClassVar, Type
+import pytest
+
+from orion.benchmark.task.base import BaseTask
+from orion.benchmark.task.profet.profet_task import (
+    MetaModelTrainingConfig,
+    ProfetTask,
+    download_data,
+    load_data,
+)
+from orion.algo.space import _Discrete
+from typing import Dict, Tuple
+import numpy as np
+import torch
+
+from logging import getLogger as get_logger
+from .conftest import REAL_PROFET_DATA_DIR, requires_profet_data
+
+logger = get_logger(__name__)
+shapes: Dict[str, Tuple[Tuple[int, ...], Tuple[int, ...], Tuple[int, ...]]] = {
```

Comment:
It would be better if we can separate these by module.

Reply:
The `load_fake_data` fixture below needs access to many different stats (shape, mean, min, max) for all the benchmarks. Do you have a suggestion of how to do this cleanly if splitting these dicts between the modules?

Reply:
Fixed.